all/test/all_test.go

```diff
@@ -688,6 +688,17 @@ func (s *TestSuite) TestAllCases() {
 				defer os.RemoveAll(dir)
 				s.Require().NoError(err)
 				err = copy.Copy(src, path.Join(dir, "all"), opt)
+
+				// We previously had a buggy PYTHONPATH which would search the working directory for
+				// modules when running the Python generator.
+				//
+				// Let's add something that with the same name as a standard library module to the test
+				// directory as a regression test.
+				//
+				// Further context: https://github.com/namely/docker-protoc/pull/356
+				_, err = os.OpenFile(path.Join(dir, "io.py"), os.O_RDONLY|os.O_CREATE, 0666)
```

Comment:
I believe each test gets executed with PWD as the temporary dir created just above though.

Reply:
> Sorry for the delay

No worries!

> Right, but we copy the all directory to the temp directory, so I think if you create the file before copying it, I think it should work.

But the file will end up in $pwd/all rather than $pwd which is required to trigger this bug. I could add an extra line to also copy `io.py` as well but it will need to go in this spot.